webpack/webpack.b3.common.js

```diff
@@ -14,6 +14,7 @@ module.exports = Object.assign({}, commonDefault, {
         new webpack.ProvidePlugin({
             '$': 'jquery',
             'jQuery': 'jquery',  // needed for bootstrap 3 to work
+            'window.jQuery': 'jquery',
```

Comment:
Does this mean webpack is making jQuery globally available? Why is it needed? Would it make sense to add a comment here about that?

Is it possibly related to `quicksearch`, which is removed in a later commit in this PR?

The same applies to `webpack/webpack.common.js`

Reply:
Yes, it does. The two previous lines do something pretty similar by automatically importing jQuery everywhere. It's needed because the multiselect library expects `window.jQuery`: https://github.com/lou/multi-select/blob/57fb8d3f5d27a0e1058f63921cdb26b4d30da361/js/jquery.multi-select.js#L544